src/components/customer-resource-show.js

```diff
@@ -76,6 +78,16 @@ export default function CustomerResourceShow({ customerResource, toggleRequest,
                   </div>
                 </KeyValueLabel>
               ) }
+              {record.managedCoverageList && record.managedCoverageList.length > 0 && isValidCoverageList(record.managedCoverageList) && (
```

Comment:
`record.managedCoverageList.length > 0` can just be `record.managedCoverageList.length` since `0` is falsey

Reply:
Great - thank for catching that!